docs/docs/cli/integrations.md

```diff
@@ -0,0 +1,42 @@
+
+## Overview
+
+Mailscript provides integrations with the external services Google Drive and Zoom. These integrations provide validated OAuth tokens to Mailscript Actions after the integrations are setup. The provided OAuth tokens can be used to interact directly with the integrated service.
+
+## Google Drive
+
+The Google Drive Integration provides an OAuth token with the permissions defined by [drive.files](https://www.googleapis.com/auth/drive.file). This allows for viewing and managing Google Drive files and folders that you have opened or created with this app.
+
+### Setup
+
+To setup the Google Drive Integration, simply ensure you are logged in under the CLI with:
+```
+mailscript login
+```
+
+Then, add the Google Drive Integration with the following:
+```
+mailscript integrations:add --gdrive
+```
+
+This will open up your default web browsers with a prompt to sign in to Google.
+
+### Usage
+
+To access the Google Drive Integration from an Action, [Variable Interpolation]() is required on `integrations.google`, which returns the Integration's OAuth token. 
+
+Note: The Integration's OAuth token is only provided to Webhook-typed Actions.
+
+## Zoom
+
+The Zoom Integration provides an OAuth token. This allows for full access to the Zoom API with your authority.
+
+Note: This Integration is not currently supported.
+
+### Setup
+
+There is currently no easy way to setup the Zoom Integration and is not currently supported.
```

Comment:
Maybe replace this line with coming soon or similar?

Reply:
Not sure if this is coming soon. Last I heard, there aren't plans on going further with it for the time being.

Reply:
So maybe just remove the mention of Zoom entirely if it not something the user can use.

Reply:
It is accessible thru the API tho

Reply:
@Geo25rey I would suggest adding some mention of the API in that case otherwise it would appear like we are offering a feature that isn't quite ready yet. Better to not confuse the reader with this.